### ISSUE TITLE
ESI routes that return 204 throw "StatusError: No Content"

### ISSUE THREAD

**Issue by diane1f0cd** [MichielvdVelde/eve-esi#6]
Valid status codes are explicitly set to 200, and 201 for non-GET requests, here: https://github.com/MichielvdVelde/eve-esi/blob/fff168f17b28fed7c7efa505795ac1c8929e7842/src/index.ts#L207

However, some ESI endpoints return a status code of 204 upon success (e.g. PUT routes like `/characters/{character_id}/calendar/{event_id}/`, every POST route under `/ui/`, every DELETE route).

**Comment by MichielvdVelde** [comment]
I agree that the 204 status code should probably be added to the default options for non-GET requests. I will make the modification in an upcoming maintenance release.

Thanks for bringing this to my attention!
